droidlet/dialog/droidlet_nsp_model_wrapper.py

```diff
@@ -289,30 +296,24 @@ def handle_logical_form(
         """Return the appropriate DialogueObject to handle an action dict d
         d should have spans filled (via process_spans_and_remove_fixed_value).
         """
-        ProgramNode.create(self.agent.memory, logical_form)
-
+        ProgramNode.create(self.dialogue_manager.memory, logical_form)
+        dop = self.dialogue_object_parameters
```

Comment:
nit: can we name the variable `dialogue_object_parameters` or something else more informational ?

Reply:
fixed in c576a26cca76c32b5ce4fdfd9d89faec68c9d96b